.github/workflows/dependency-review.yaml

```diff
@@ -12,6 +12,7 @@ on:
   pull_request:
     branches:
       - main
+      - master
```

Comment:
This should be handled on the calling repo, not this repo. Look at how this is called on teleport repo.

Reply:
Ok, I was trying to see if we could reduce how many repo's needed the branch definition.  If we want it to be defined on the calling repo always should I remove the branches entirely?